src/navigation_controller.ts

```diff
@@ -12,43 +12,43 @@
 
 import './gesture_monkey_patch';
 
-import {ASTNode, ShortcutRegistry} from 'blockly/core';
+import * as Blockly from 'blockly/core';
+import {ASTNode, ShortcutRegistry, BlockSvg, WorkspaceSvg} from 'blockly/core';
 import {utils as BlocklyUtils} from 'blockly/core';
 
 import * as Constants from './constants';
 import {Navigation} from './navigation';
 import {Announcer} from './announcer';
+import {LineCursor} from './line_cursor';
+import {BlockCopyData} from 'node_modules/blockly/core/clipboard/block_paster';
```

Comment:
Can you add a TODO with a tag for the issue you filed?

Reply:
Done